.drone.yml

```diff
@@ -80,372 +77,340 @@ steps:
+  - name: Create Kind Cluster
+    image: quay.io/sighup/dind-kind-kubectl-kustomize:0.20.0_1.29.1_3.10.0
     pull: always
-    volumes:
-      - name: shared
-        path: /shared
-    depends_on: [init]
+    environment:
+      CLUSTER_NAME: ${DRONE_REPO_NAME}-${DRONE_BUILD_NUMBER}-1.26.6
+      KIND_CONFIG: ./config-$${CLUSTER_NAME}.yaml
+      CLUSTER_VERSION: 1.26.6
+      KUBECONFIG: ./kubeconfig-$${CLUSTER_NAME}
     commands:
-      - export CLUSTER_NAME=124
-      - export INSTANCE_IP=$(cat /shared/machine/ip)
-      - export KUBECONFIG=/shared/kube/kubeconfig-125
-      - bats -t katalog/tests/tests.bats
-
-  - name: destroy
-    image: quay.io/sighup/e2e-testing-drone-plugin:v1.25.3
+      - kind create cluster --config $${KIND_CONFIG}
+      - kind get kubeconfig --name $${CLUSTER_NAME} > $${KUBECONFIG}
+    depends_on:
+      - Generate Kind Config file
+  - name: End-to-End Tests
+    image: quay.io/sighup/e2e-testing:1.1.0_0.11.0_3.1.1_1.9.4_1.26.3_3.5.3_4.33.3
     pull: always
-    depends_on: [e2e]
-    settings:
-      action: destroy
-      pipeline_id: cluster-125
-      aws_default_region:
-        from_secret: aws_region
-      aws_access_key_id:
-        from_secret: aws_access_key_id
-      aws_secret_access_key:
-        from_secret: aws_secret_access_key
-      terraform_tf_states_bucket_name:
-        from_secret: terraform_tf_states_bucket_name
-      vsphere_server:
-        from_secret: vsphere_server
-      vsphere_password:
-        from_secret: vsphere_password
-      vsphere_user:
-        from_secret: vsphere_user
-      dockerhub_username:
-        from_secret: dockerhub_username
-      dockerhub_password:
-        from_secret: dockerhub_password
+    network_mode: host
+    environment:
+      CLUSTER_NAME: ${DRONE_REPO_NAME}-${DRONE_BUILD_NUMBER}-1.26.6
+      KUBECONFIG: ./kubeconfig-$${CLUSTER_NAME}
+    depends_on:
+      - Create Kind Cluster
+    commands:
+      - export INTERNAL_PORT=$(grep 'internalPort' ./env-$${CLUSTER_NAME}.yaml | awk '{ print $2 }')
+      - export EXTERNAL_PORT=$(grep 'externalPort' ./env-$${CLUSTER_NAME}.yaml | awk '{ print $2 }')
+      - until kubectl get serviceaccount default > /dev/null 2>&1; do echo "waiting for control-plane" && sleep 1; done
+      - bats -t ./katalog/tests/tests.bats
+  - name: Destroy Kind Cluster
+    image: quay.io/sighup/dind-kind-kubectl-kustomize:0.20.0_1.29.1_3.10.0
+    environment:
+      CLUSTER_NAME: ${DRONE_REPO_NAME}-${DRONE_BUILD_NUMBER}-1.26.6
+    commands:
+      - "kind delete cluster --name ${CLUSTER_NAME} || true"
+    depends_on:
+      - End-to-End Tests
     when:
       status:
         - success
         - failure
-
-volumes:
-  - name: shared
-    temp: {}
-
 ---
-name: e2e-kubernetes-1.26
 kind: pipeline
 type: docker
-
-depends_on:
-  - policeman
-
-node:
-  runner: internal
-
+name: E2E Tests on kind clusters version 1.27.3
 platform:
+  CLUSTER_NAME: ${DRONE_REPO_NAME}-${DRONE_BUILD_NUMBER}-1.26.6
```

Comment:
I think this line does **not** belong here, and if it does the version is wrong, it should be 1.27.3
```suggestion
```